.helm/ecamp3/templates/print_secrets.yaml

```diff
@@ -12,6 +12,14 @@ data:
   sentry-dsn: ""
   {{- end }}
 
+  {{- if .Values.browserless.enabled }}
+  browser-ws-endpoint: {{  printf "ws://%s:%d" (include "browserless.name" .) .Values.browserless.service.port | b64enc | quote }}
+  {{- else if .Values.print.browserWsEndpoint }}
+  browser-ws-endpoint: {{ .Values.print.browserWsEndpoint | b64enc | quote }}
+  {{- else}}
+  browser-ws-endpoint: ""
+  {{- end }}
+
   {{- if .Values.print.browserWsEndpoint }}
   browser-ws-endpoint: {{ .Values.print.browserWsEndpoint | b64enc | quote }}
   {{- else}}
```

Comment:
Remove, you replaced this with the above new logic

Reply:
Thanks. Fixed in https://github.com/ecamp/ecamp3/pull/2656/commits/c0370bc8f32d9ed0a00f3ac156f33a012c39cf36